lib/resolve-value.js

```diff
@@ -5,6 +5,29 @@ var gatherVariableDependencies = require('./gather-variable-dependencies');
 var findNodeAncestorWithSelector = require('./find-node-ancestor-with-selector');
 var cloneSpliceParentOntoNodeWhen = require('./clone-splice-parent-onto-node-when');
 
+/*
+	Object.assign polyfill
+	https://gist.github.com/WebReflection/10404826
+*/
+if (!('assign' in Object)) {
+  Object.assign = function(has){
+    'use strict';
+    return assign;
+    function assign(target, source) {
+      for (var i = 1; i < arguments.length; i++) {
+        copy(target, arguments[i]);
+      }
+      return target;
+    }
+    function copy(target, source) {
+      for (var key in source) {
+        if (has.call(source, key)) {
+          target[key] = source[key];
+        }
+      }
+    }
+  }({}.hasOwnProperty);
+}

```

Comment:
What are you trying to support with this?

This polyfill doesn't seem necessary for Node v4+, https://kangax.github.io/compat-table/es6/#test-Object_static_methods

Better to pull in a separate package from npm if we actually do want this.